### PR TITLE
Feat/js

### PR DESCRIPTION
**Pull request by Blu-J**
### About

Update the version and bring in the js

### Proof

Showing health check running, get/ set config, and seeing the properties
https://user-images.githubusercontent.com/2364004/179061945-8dc89e03-70fc-449f-9d20-168481107198.mp4

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,26 +19,25 @@ verify: vaultwarden.s9pk $(S9PK_PATH)
...
-vaultwarden.s9pk: manifest.yaml LICENSE image.tar instructions.md icon.png $(ASSET_PATHS)
+vaultwarden.s9pk: manifest.yaml LICENSE image.tar instructions.md icon.png $(ASSET_PATHS) scripts/embassy.js
 	embassy-sdk pack
 
 instructions.md: docs/instructions.md $(DOC_ASSETS)
 	cd docs && md-packer < instructions.md > ../instructions.md
 
 image.tar: Dockerfile $(VAULTWARDEN_SRC) docker_entrypoint.sh check-web.sh manifest.yaml
 	cp ./docker_entrypoint.sh ./vaultwarden/docker_entrypoint.sh
-	cp ./check-web.sh ./vaultwarden/check-web.sh
```
Comment (elvece) [inline Makefile:30]: why did we need to remove this?

File: manifest.yaml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 id: vaultwarden
 title: Vaultwarden
-version: 1.22.2
-release-notes: "Upgrade to EmbassyOS v0.3.0 and support project rename to Vaultwarden. Visit here for more information: https://github.com/dani-garcia/vaultwarden/discussions/1642"
-license: gnu
+version: 1.25.0
+release-notes: |
+  * Updates to new version of Vaultwarden to 2.25.0. Lots of bug fixes + features. Look at https://github.com/dani-garcia/vaultwarden/releases/tag/1.25.0 for more
```
Comment (elvece) [inline manifest.yaml:5]:
```suggestion
  * Updates to Vaultwarden v2.25.0. Lots of bug fixes + features. Look at https://github.com/dani-garcia/vaultwarden/releases/tag/1.25.0 for more
```

File: scripts/services/health.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const guardDurationAboveMinimum = (
+  input: { duration: number; minimumTime: number },
+) =>
+  (input.duration <= input.minimumTime)
+    ? Promise.reject(errorCode(60, "Starting"))
+    : null;
+
+const checkHealthDuration = 5000;
+
+export const healtAlive: T.ExpectedExports.health[""] = async (
```
Comment (elvece) [inline scripts/services/health.ts:29]:
```suggestion
export const healthAlive: T.ExpectedExports.health[""] = async (
```

File: scripts/services/health.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  }
+  return error("Web interface is unreachable");
+};
+
+/** These are the health checks in the manifest */
+export const health: T.ExpectedExports.health = {
+  /** Checks that the server is running and reachable via cli */
+  // deno-lint-ignore require-await
+  async alive(effects, duration) {
+    return healtAlive(effects, duration).catch(catchError(effects));
```
Comment (elvece) [inline scripts/services/health.ts:49]:
```suggestion
    return healthAlive(effects, duration).catch(catchError(effects));
```

**Review by elvece — CHANGES_REQUESTED** [review]
Few suggestions, health check implementation is awesome 🎉

File: Makefile (hunk 1 above)
Comment (Blu-J) [reply Makefile:30]: Because now we are using the js version of the health checks

Comment (Blu-J) [reply Makefile:30]: Also, going to remove the file too since we don't need

**Review by elvece — APPROVED** [review]

**Pull request merged**